react/week2/todolistsapp/src/components/todosCollection.js

```diff
@@ -0,0 +1,14 @@
+export const todos = [
```

Comment:
Since `todosCollection` is not a component, it's better to keep it out of components folder, and for example in a folder called `data`